<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>229</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A4 to F4 Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/favicon.png</normaloff>:/img/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2" colspan="2">
     <widget class="QLabel" name="output_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="output_label">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="img2_button">
      <property name="minimumSize">
       <size>
        <width>127</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Browse File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="start">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="img2_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="img2_label">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Image 2:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="git">
      <property name="minimumSize">
       <size>
        <width>46</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Made With ðŸ’– By &lt;/span&gt;&lt;a href=&quot;https://github.com/srt19&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Srt19&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="help_button">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>29</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/help_light.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="img1_button">
      <property name="minimumSize">
       <size>
        <width>127</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Browse File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="img1_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="output_button">
      <property name="minimumSize">
       <size>
        <width>127</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Browse File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="img1_label">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Image 1:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="5" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;READY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="lang_button">
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>EN</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
